common.gradle

```diff
@@ -189,17 +189,12 @@ dependencies {
     }
 
     arImplementation 'com.google.protobuf:protobuf-lite:3.0.0'
-	if (file("../../../SXR").exists()) {
-        if (project.hasProperty("useLocalDependencies") && project.useLocalDependencies) {
-            if (findProject(':sxr_adapter')) {
-                arImplementation project(':sxr_adapter')
-            } else {
-                arImplementation(':sxr_adapter-release@aar')
-            }
+    if (project.hasProperty("useLocalDependencies") && project.useLocalDependencies) {
```

Comment:
What is the purpose of this change?

Reply:
@liaxim This change is not required. I have reverted